src/main/charts/bamboo/values.yaml

```diff
@@ -506,7 +522,7 @@ bamboo:
     
     # -- How often (in seconds) the Bamboo container readiness probe will run
     #
-    periodSeconds: 5
+    periodSeconds: 10
```

Comment:
Bamboo does not like to have its `/status` endpoint hit on a regular basis otherwise issues can occur during setup. Increase this threshold to mitigate this issue being encountered.